app/models/package.rb

```diff
@@ -0,0 +1,10 @@
+###
+#  Class to connect to the VND_PACKAGES table
+###
+class Package < ActiveRecord::Base
+  self.table_name = 'vnd_packages'
```

Comment:
I think the eloader vendor table is called vnd_packages_gen or something like that. Vnd_packages contains all the non eloader vendors.

Reply:
You’re right. Sorry! I was going off of my memory. I think I was confusing it with the table that holds the files.